deployment/templates/mongodb/mongodb-deployment.yaml

```diff
@@ -14,7 +14,7 @@ spec:
       labels:
         app: {{ .Values.mongodb.appName }}
     spec:
-      {{ if eq .Values.clusterType "kubernetes" }}
+      {{ if and (eq .Values.clusterType "kubernetes") (eq ( .Values.mongodb.securityContext.runAsUser | int) 0) }}
```

Comment:
This disables the innit container when the security context is not activated. Does this affect you @zagganas  ?

Reply:
Probably not. If it does, I guess I could still run everything as root.